test/journeys/specs/space/data-api.js

```diff
@@ -1,16 +1,22 @@
 import testUsers from '@ciscospark/test-helper-test-users';
-import '@ciscospark/internal-plugin-conversation';
 import '@ciscospark/plugin-logger';
 import CiscoSpark from '@ciscospark/spark-core';
+import '@ciscospark/internal-plugin-conversation';
 
-import {switchToMeet} from '../../../lib/test-helpers/space-widget/main';
-import {updateJobStatus} from '../../../lib/test-helpers';
-import {elements, declineIncomingCallTest, hangupBeforeAnswerTest, hangupDuringCallTest} from '../../../lib/test-helpers/space-widget/meet';
+import {jobNames, updateJobStatus} from '../../lib/test-helpers';
+import waitForPromise from '../../lib/wait-for-promise';
+import {switchToMeet} from '../../lib/test-helpers/space-widget/main';
+import {elements, declineIncomingCallTest, hangupBeforeAnswerTest, hangupDuringCallTest} from '../../lib/test-helpers/space-widget/meet';
+import {
+  sendMessage,
+  verifyMessageReceipt
+} from '../../lib/test-helpers/space-widget/messaging';
 
-describe('Widget Space: Data API', () => {
+describe('Space Widget Data API Tests', () => {
   const browserLocal = browser.select('browserLocal');
   const browserRemote = browser.select('browserRemote');
-  const jobName = 'react-widget-space-dataApi';
+  const spaceWidget = '.ciscospark-space-widget';
+  const jobName = jobNames.space;
```

Comment:
I'm using the same name for all the space widget tests, should I change them to be more file specific?